templates/questions/index.html.ep

```diff
@@ -36,7 +36,6 @@ $(document).ready(function() {
 	
     setCurrentUser("<%=session 'username'%>");
     setAdmin("<%=session 'admin'%>");
-    setVoteFloor("<%= $self->config->{vote_floor}%>");
```

Comment:
This was used in the javascript to detect when to play the delete animation for a question when it was beneath this value. Because the questions are no longer deleted, the variable is no longer used in the javascript so we removed it.